test/clj/rems/test_pdf.clj

```diff
@@ -40,36 +40,54 @@
                    (#'pdf/render-fields data)))))))
     (is (some? (with-language :en #(pdf/application-to-pdf-bytes data))))))
 
-;; reviewer should not be able to see applicant private form fields
-(deftest test-pdf-reviewer-private-fields
+(deftest test-pdf-privacy
```

Comment:
I feel like `test-pdf-private-form-fields` would be a better name, but that is nitpicking, feel free to either correct it or leave as it is.